cpp/tests/spatial/hausdorff_toy.cu

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <time.h>
+#include <sys/time.h>
+#include <vector>
+#include <string>
+#include <iostream>
+
+#include <thrust/device_vector.h>
+
+#include <utilities/error_utils.hpp>
+#include <gtest/gtest.h>
+
+#include <tests/utilities/cudf_test_utils.cuh>
+
+#include <cuspatial/soa_readers.hpp> 
+#include <cuspatial/hausdorff.hpp> 
+
+#include <tests/utilities/column_wrapper.cuh>
+#include <tests/utilities/cudf_test_utils.cuh>
+#include <tests/utilities/cudf_test_fixtures.h>
+
+using namespace cuspatial;
+
+struct HausdorffToy : public GdfTest 
```

Comment:
Why is it called a Toy? Please don't call them toys. Users need to know that cuSpatial is for real use.

Reply:
It means that that the test uses simple toy data just for demonstration purposes - some packages use the convention. Please feel free to change.